omero/developers/Java.rst

```diff
@@ -339,6 +339,25 @@ This is useful when you need the pixels intensity.
         store.close();
     }
 
+-  **Retrieve a histogram.**
+
+::
+
+    RawPixelsStorePrx store = gateway.getPixelsStore(ctx);
```

Comment:
in the example you have the store initialised in the ``try/catch``
you should unify that

Reply:
And I just noticed that the getHistogram method is already in the RawDataFacility, which should be used preferably. I have to update the example and doc again.